ios/GoodDollar/DeepLink+AppDelegate.m

```diff
@@ -8,13 +8,24 @@
 #import <Foundation/Foundation.h>
 #import "AppDelegate.h"
 #import <RNBranch/RNBranch.h>
+#import <RNTorusDirectSdk/RNTorus.h>
```

Comment:
@MarianoAmado 

1. let's enhance code a bit

```objectivec
// Add the openURL and continueUserActivity functions
- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
    NSString *urlString = url.absoluteString;
   
    // if url processed by RNBranch - return
    if ([RNBranch.branch application:app openURL:url options:options]) {
        return YES;
    }

    // Facebook, google, other OAuth handlers could be here

    // if it's not torus redirect - return
    if (![url.host isEqualToString:@"org.gooddollar"] || ![url.path isEqualToString:@"/redirect"]) {
        return YES;
    }
        
    // handle torus redirect: "gooddollar://org.gooddollar/redirect"
    if (@available(iOS 11.0, *)) {
        [RNTorusDirectSdk handle:urlString];
    }

    return YES;
}

- (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray * _Nullable))restorationHandler {
  // handler for Universal Links
  [RNBranch continueUserActivity:userActivity];

  return YES;
}
```

2. how about IOS < 11 ? how torus could handle redirect in that case ? You have commented 'Fallback on earlier versions' - does the solution for this fallback exists ? Your just haven't implemented it yet or you're awaiting it from the Torus team ?

Reply:
1. there's no need for this refactoring.
if at all the handling of torus redirect should be first and not last, because it is more specific.
2. @MarianoAmado i've merged this, but please comment

Reply:
@sirpy @serdiukov-o-nordwhale 
This is because their native module doesn't support lower ios versions (we could ask them about supporting v10 but I think there's a reason for that)

Maybe we should disable the torus screens when detecting lower ios versions
(and/or add some message about it like in `components/browserSupport`)